force-app/main/default/lwc/attendanceRow/attendanceRow.html

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div
+        class="slds-box small-box slds-var-m-around_x-small slds-var-p-horizontal_medium"
+    >
+        <lightning-record-edit-form
+            record-id={recordId}
+            object-api-name={serviceDeliveryObject}
+        >
+            <!-- onload={handleLoad}
+                onsuccess={handleSuccess}
+                onsubmit={handleSubmit}
+                onerror={handleSaveError} -->
+            <lightning-layout multiple-rows="true" horizontal-align="left">
+                <template for:each={fieldSet} for:item="field">
+                    <lightning-layout-item
+                        size="12"
+                        small-device-size="6"
```

Comment:
I wonder if we want to try and anchor the fields to the bottom or truncate the label or something, when you shrink the screen the wrapping of text on one field makes the alignment off on the others. Let me know what you think... 
![Screen Shot 2020-11-06 at 8 26 01 AM](https://user-images.githubusercontent.com/450473/98376986-e002b380-2009-11eb-8ecd-e2d4bc08f1ae.png)

Reply:
Thumbs up for this idea - would be cool to find a way to keep things looking nice, even when compact.